.github/guides/SMOOTHNESS.md

```diff
@@ -0,0 +1,75 @@
+## Guide to Smooth Visuals
+
+I'd like to begin this document with two disclaimers:
+- This is my view of the game/engine at this precise moment in time. Whenever you write something down it will always rot.
+- This is far more information about how to make BYOND behave then you likely need. I'm writing it down because it's important that it's documented, because otherwise our solutions would be absolutely incomprehensible.
+
+### What does smooth mean?
+
+The game server fires at a set tickrate (20 tps or 0.5ds tick_lag on mainline /tg/). Clients are sent things like map data, animations, etc once each tick, but have their own independant tick rate (100 tps by default, this should be divisable by the server's tick rate). Each client tick animations will continue on along some gradient between the start and end values, etc etc.
+
+Because the client and server are not well, the same thing, they have their own opinions about how far along in an animation the client is in any given frame. Our goal here is to minimize this difference in opinion, to keep the two as synced together as possible.
+
+We're doing this because if we can keep them in time, animations/effects will transition smoothly, and we avoid "jitter" in the client's smooth interpolation.
+
+Achieving smoothness has other benefits, this idea of client and server desyncs applies to stuff like icon state animations and sound lengths just the same.
+
+### What causes this difference in opinion?
+
+> In order to talk about this properly you need to have a basic understanding of the game tick. See this [writeup on tick order](TICK_ORDER.md) for a baseline.
+
+As discussed before the client and server fire at different tick rates. What this means mechanically is the server will try to send out a packet of information to the client every tick_lag ds, and the client will anticipate a packet of information from the server every tick_lag ds. If the client recieves its packet on regular intervals, and is able to render at its specified framerate, then any animations or effects will happen smoothly, assuming we don't literally fuck up the timing on the server.
+
+There's a bunch of different things that can happen which can fuck up this cycle.
+
+#### 1. Network Lag
+
+A series of tubes separates the client from the server, and if the packet of information containing appearance updates gets slowed down in transit then we'll see that as jitter on the client.
+
+There isn't a ton we can actually do about this, because sometimes the internet fucks up. We kind of just have to assume the network connection will be stable.
```

Comment:
```suggestion
There isn't a ton we can actually do about this, because sometimes the internet fucks up. We operate under the assumption the network connection will be stable.
```

Reply:
Seems more clunky to me, is this understandable as is?